match/lib/match/export.rb

```diff
@@ -0,0 +1,132 @@
+module Match
+  class Export
+    attr_accessor :params
+    attr_accessor :type
+
+    attr_accessor :certs
+    attr_accessor :keys
+    attr_accessor :profiles
+    attr_accessor :files
+
+    def run(params, type: nil)
+      self.params = params
+      self.type = type
+      params[:workspace] = GitHelper.clone(params[:git_url], params[:shallow_clone])
+      FastlaneCore::PrintTable.print_values(config: params,
```

Comment:
This file seems like a lot of duplicate code from what we already have in `match`, right?